UIComponentsExample/UIComponentsExample/UIComponentsExample.entitlements

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>com.apple.security.app-sandbox</key>
+    <true/>
+    <key>com.apple.security.files.user-selected.read-only</key>
+    <true/>
+</dict>
+</plist>
```

Comment:
why?

Reply:
No idea, I think it got added to the project automatically. Im not sure what its doing, should it be deleted?

Reply:
Try to delete and see if it works

Reply:
I just tried to delete it, im not able to build without it 
<img width="1314" alt="image" src="https://user-images.githubusercontent.com/32563707/219331356-259b0475-1e6d-447c-83c6-7eefd4f4066d.png">